test/tests/button/funding.js

```diff
@@ -574,14 +574,14 @@ describe(`paypal button component funding mix`, () => {
         }, '#testContainer');
     });
 
-    it('should render elv in vertical layout', (done) => {
+    it('should not render elv in vertical layout', (done) => {
```

Comment:
should this be "horizontal"?

Reply:
Hah, my bad! This is what peer reviews are for.